source/gsoc.md

```diff
@@ -124,36 +123,6 @@ Each of the proposed subjects (link to be provided if we are confirmed as a
 mentoring organisation) applies to a single ASF project. You will need to
 engage with that project community.
 
-<a name="GSoC-ProspectiveASFmentors:readthis"></a>
-<a name="formentors"></a>
-# Prospective ASF mentors: read this
-
-We are looking for as many interesting projects as we can come up with. For
-more info about mentoring, please read our [guide to being a mentor](guide-to-being-a-mentor.html)
-.
-
-Prospective mentors must join the mentors@community.apache.org mailing list,
-this is where mentor specific issues are dealt with, and where
-announcements will be made. If you want to track the program
-administration you should subscribe to dev@community.apache.org.
-
-Once the ASF is confirmed as a mentoring organisation mentors must register
-with the GSoC webapp, and request to become a mentor for the ASF
-organization. Make sure that the email address you use for that (it's often
-your @gmail.com address by default) is '''registered as a mail alias for your Apache account at (https://id.apache.org)''' so that we can match it to your ASF account.
-
-All ASF projects are invited to submit their ideas to their issue tracker,
```

Comment:
@priyasharma1 this part is removed completely :(
Could you please restore it?

Reply:
Hi @solomax ,
The "Summary" heading on the mentor's guide page has similar information about JIRA and labels, which is why I didn't add it again. Do you want me to keep this specific information as is on this page?

Or you meant to restore the entire removed section?

Reply:
Hello @priyasharma1,

sorry for the delay :)

the paragraph here clearly states 

 `please be sure to add the labels "*gsoc2023*" and "*mentor*" so that we can automatically include them in our list of subjects` the text in mentor is less clear (at least for me)
 
The text at `guide-to-being-a-mentor.md` is very much detailed, so it's hard for me to catch required steps

maybe we can style important/required parts somehow ... 
not sure

Reply:
Hi @solomax,
I understand your concern, please refer to the following screenshot and let me know if this is clear enough.

![Screenshot from 2023-02-27 12-20-27](https://user-images.githubusercontent.com/14103632/221494526-e6786fe0-2201-4787-b76e-3c473901dca6.png)

Reply:
LGTM!
We can add additional bootstrap styling later :)

Reply:
Great!
I have updated the PR.